archive/tar_windows.go

```diff
@@ -302,15 +303,14 @@ func tarToBackupStreamWithMutatedFiles(buf *bufio.Writer, w io.Writer, t *tar.Re
 	return writeBackupStreamFromTarFile(buf, t, hdr)
 }
 
-// writeBackupStreamFromTarFile writes a Win32 backup stream from the current tar file. Since this function may process multiple
-// tar file entries in order to collect all the alternate data streams for the file, it returns the next
-// tar file that was not processed, or io.EOF is there are no more.
-func writeBackupStreamFromTarFile(w io.Writer, t *tar.Reader, hdr *tar.Header) (*tar.Header, error) {
-	bw := winio.NewBackupStreamWriter(w)
+// reads the SDDL associated with the header of the current file
+// from the tar header and encodes it into a byte array.
+func encodeSDDLFromTarHeader(hdr *tar.Header) ([]byte, error) {
```

Comment:
I suppose we'll swap to the go-winio versions of these when ready?

Reply:
Yep. Waiting to merge this PR: https://github.com/microsoft/go-winio/pull/220